Sources/EasyPurchase/ProductInfoController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ProductInfoController.swift
+//
+//
+//  Created by BJIT on 19/9/23.
+//
+import Foundation
+import StoreKit
+
+// A protocol for handling product information retrieval.
+protocol ProductsInfoHandler {
```

Comment:
If this protocol only confirmed by apple the make it a class or any object type.

Reply:
Made it to AnyObject type. Thank you.